package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "typeorm",
   "private": true,
-  "version": "0.0.9",
+  "version": "0.1.0-alpha.1",
```

Comment:
> Using the caret for versions less than 0.1.0 offers no flexibility at all. Only the exact version specified will be valid. 
For example, ^0.0.3 will only permit only exactly version 0.0.3.

> For versions greater than or equal to 0.1.0, but less than 1.0.0, the caret adopts the same behaviour as a tilde and will allow flexibility in patch versions (only).
For example, ^0.1.3 will permit all versions from 0.1.3 to the next minor, 0.2.0.

So if you decide to go to 0.1.0, from now you have to keep semver, so 0.1.1 has to be minor release with no breaking changes. If you want to keep active development and introduce many changes, I would recommend to stick to 0.0.x because you don't have to worry about breaking changes - ^0.0.9 will install only 0.0.9, not the breaking things release 0.0.10 😉 Or you might have 0.23.0 before 1.0.0 or big, rarely updates with cumulative breaking changes.